src/components/visualization/plotlyGraph/plotlyGraph.types.ts

```diff
@@ -0,0 +1,13 @@
+import type {Updates, VisualizationFormat} from "../../types";
+import type {PlotParams} from "react-plotly.js";
+
+
+export type PlotlyJson = Omit<PlotParams, "divId">
+
+export interface PlotlyGraphFormat extends VisualizationFormat {
+    /** Follow [Plotly's documentation](https://plotly.com/javascript/reference/index/) to provide a valid Plotly JSON.
+     * Note: It excludes the `divId` field **/
```

Comment:
Was showing `*` after `field`
<img width="369" alt="Screenshot 2024-09-06 at 10 20 01 AM" src="https://github.com/user-attachments/assets/85bbbccc-2d58-4527-8f71-e16d4e76c673">

```suggestion
     * Note: It excludes the `divId` field */
```